### PR TITLE
Changelog for 1.16.9 (#20059)

### PR DESCRIPTION
**Pull request by 6543**
Frontport #20059

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.16.9](https://github.com/go-gitea/gitea/releases/tag/1.16.9) - 2022-06-20
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.16.9](https://github.com/go-gitea/gitea/releases/tag/v1.16.9) - 2022-06-20
```

**Review by delvh — APPROVED** [review]

**Comment by 6543** [comment]
ready once release got tagged ...

**Review by Gusted — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.16.9](https://github.com/go-gitea/gitea/releases/tag/v1.16.9) - 2022-06-20
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.16.9](https://github.com/go-gitea/gitea/releases/tag/v1.16.9) - 2022-06-21
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ This changelog goes through all the changes that have been made in each release
...
+
+* BUGFIXES
+  * Fix permission check for delete tag (#19985) (#20001)
```
Comment (6543) [inline CHANGELOG.md:11]:
```suggestion
  * Release page show all tags in compare dropdown (#20070) (#20071)
  * Fix permission check for delete tag (#19985) (#20001)
```

**Comment by 6543** [comment]
:rocket:

**Pull request merged**